### PR TITLE
Perf subdescription triangle

### PR DESCRIPTION
**Pull request by MackJoly**
* **Please check if the PR fulfills these requirements**
- [X] The commit messages follows our guidelines (explicit, in english)
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [X] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix:
- [ ] New features:
- [ ] Documentation
- [X] Optimization

* **Other information**:

### CODE REVIEW

File: volmdlr/faces.py [hunk 1]
```diff
@@ -6466,78 +6466,86 @@ def rotation_inplace(self, center: volmdlr.Point3D, axis: volmdlr.Vector3D,
...
+            else:
+                v_int = point_v1 - p0_k
+                v_int.normalize()
+                step_in = l_int / (nb_int - 1)
+                for i in range(nb_int):
+                    l = min(i * step_in, l_int)
+                    if l != 0:
+                        p1_res.append(p0_k + v_int * l)
+
+        return npy.unique(p0_res + p1_res).tolist()
```
Comment (WirajanDASILVA) [inline volmdlr/faces.py:6515]: Could you please try renaming some variables while being more explicit in what each variable is?

File: volmdlr/faces.py [hunk 2]
```diff
@@ -6466,78 +6466,86 @@ def rotation_inplace(self, center: volmdlr.Point3D, axis: volmdlr.Vector3D,
...
-
-        sub_triangles2d = [t_poly2d]
-        done = False
         while not done:
-            triangles2d = []
-            for t, subtri in enumerate(sub_triangles2d):
-                ls_length = [ls.length() for ls in subtri.line_segments]
-                ls_max = max(ls_length)
+            triangles = []
+            for t, subtri in enumerate(sub_triangles):
```
Comment (WirajanDASILVA) [inline volmdlr/faces.py:6527]: It seems t is no being used. If it is the case, could delete it along side with the enumerate?

**Review by WirajanDASILVA — CHANGES_REQUESTED** [review]

File: volmdlr/faces.py [hunk 3]
```diff
@@ -6466,78 +6466,86 @@ def rotation_inplace(self, center: volmdlr.Point3D, axis: volmdlr.Vector3D,
...
-
-        points = [pt.copy() for pt in self.points]
-        for pt in points_box:
-            if t_poly2d.point_belongs(pt):
-                points.append(pt.to_3d(frame.origin, frame.u, frame.v))
-            elif t_poly2d.point_over_contour(pt):
-                points.append(pt.to_3d(frame.origin, frame.u, frame.v))
-
-        return volmdlr.Vector3D.remove_duplicate(points)
+
```
[the remark this answers is not in the document]
Comment (MackJoly) [reply volmdlr/faces.py:6507]: Done

File: volmdlr/faces.py (hunk 1 above)
Comment (MackJoly) [reply volmdlr/faces.py]: I tried to change, can you confirm ?

File: volmdlr/faces.py (hunk 2 above)
Comment (MackJoly) [reply volmdlr/faces.py]: Thanks for the remark, I delete it

**Review by WirajanDASILVA — APPROVED** [review]

**Pull request merged**